infra/components/emr_serverless_analytics/__init__.py

```
@@ -0,0 +1,429 @@
+        super().__init__(
+            f"custom:emr-serverless-analytics:{name}",
+            name,
+            None,
+            opts,
+        )
+
+        region = aws.get_region().name
+        account_id = aws.get_caller_identity().account_id
+
+        # Convert inputs to Outputs
+        langsmith_bucket_name = Output.from_input(langsmith_export_bucket_name)
```

Ruff (0.14.10)

Local variable `langsmith_bucket_name` is assigned to but never used

Remove assignment to unused variable `langsmith_bucket_name`

(F841)